packages/cms-data-sync/src/run-migrations.ts

```diff
@@ -12,7 +12,7 @@ import { migrateTutorials } from './tutorials/tutorials.data-migration';
 import { logger } from './utils';
 import { contentfulRateLimiter } from './contentful-rate-limiter';
 
-export const runMigrations = async () => {
+export const runMigrations = async (flags: string[] = []) => {
```

Comment:
I would do
```suggestion
export const runMigrations = async (flags: Flag[] = []) => {
```
then then somewhere else:
```typescript
const flags = [
 "teams",
 (....)
] as const;

export type Flag = (typeof flags)[number];
```
and then in your index file you could add a very simple validation after https://github.com/yldio/asap-hub/blob/f50fdd36bd4ace1bf9efda0b6a60cab623954cdb/packages/cms-data-sync/src/index.ts#L4 to make sure that the string is then recognised as an element from the union